bundles/org.openhab.binding.ambientweather/src/main/java/org/openhab/binding/ambientweather/internal/handler/AmbientWeatherEventListener.java

```diff
@@ -272,7 +272,9 @@ private void handleSubscribed(String jsonData) {
         try {
             EventSubscribedJson subscribed = gson.fromJson(jsonData, EventSubscribedJson.class);
             if (subscribed.invalidApiKeys != null) {
-                logger.debug("Listener: Invalid keys!! invalidApiKeys={}", subscribed.invalidApiKeys.toString());
+                logger.info("Listener: Invalid keys!! invalidApiKeys={}", subscribed.invalidApiKeys.toString());
```

Comment:
There is no need to call `toString()`. The logger takes care of that.

```suggestion
                logger.info("Listener: Invalid keys!! invalidApiKeys={}", subscribed.invalidApiKeys);
```

Reply:
Thanks. Fixed.